### PR TITLE
Add missing arguments

### PR DESCRIPTION
**Pull request by mjackson**

### CODE REVIEW

File: src/component.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ const TinyRendererComponentMixin = {
     return this.node;
   },
 
-  receiveComponent() {
+  receiveComponent(nextElement, transaction, context) {
```
Comment (iamdustan) [inline src/component.js:139]: I had neglected this since I wasn’t using them in this file. I suppose it’s worth adding them, regardless, though?

Comment (mjackson) [inline src/component.js:139]: They're used below in the call to `this.updateChildren`. I was just wondering where those args came from.

Comment (iamdustan) [inline src/component.js:139]: doh

Comment (mjackson) [inline src/component.js:139]: But ya, our JSON renderer is just a single shot, so there's no real update step. I think it's useful to have the code here for documentation purposes.

**Pull request merged**